src/js/index.js

```diff
@@ -1,2 +1,49 @@
 import 'bootstrap/dist/css/bootstrap';
-import $ from 'jquery';
+
+class BitconFetcher {
+  constructor(location) {
+    this.target = location;
+  }
+
+  getPriceList(url) {
+    fetch(url)
+      .then(res => res.json())
+      .then(res => {
+        this.generateList(res);
+      })
```

Comment:
```javascript
.then(this.generateList)
```

Reply:
This is not working as soon, it seems that "this" lots its scope if we do that.